src/server.js

```diff
@@ -115,6 +115,31 @@ module.exports = function() {
 		new Identification((identHandler) => {
 			manager.init(identHandler, sockets);
 		});
+
+		// Handle ctrl+c and kill gracefully
+		let suicideTimeout = null;
+		const exitGracefully = function() {
+			if (suicideTimeout !== null) {
+				return;
+			}
+
+			// Forcefully exit after 3 seconds
+			suicideTimeout = setTimeout(() => process.exit(1), 3000);
```

Comment:
Maybe another `log.info` in there, like "Something went wrong, force-quitting now"? Also, could do `5000` just in case, but whatever.

Reply:
I don't think it's necessary to add a message.